Assets/src/WeaponReload.cs

```diff
@@ -8,51 +8,69 @@ public class WeaponReload : MonoBehaviour {
   public SoundController SoundController;
   public HUDBar ReloadBar;
   public MovementManager MovementManager;
-
-  IntervalTimer ReloadTimer;
+  private IntervalTimer ReloadTimer;
+  private const float movementSpeedReduction = 0.5f;
 
 	// Use this for initialization
 	void Start () {
-    ReloadTimer = new IntervalTimer() {
-      TimeClass = "PlayingTime"
-    };
-
-    // Reduce speed by half when reload is InProgress
+    InitializeReloadTimer();
+	}
+	
+  private void InitializeReloadTimer() {

```

Comment:
There's probably not much point in putting this in its own function if it's the only thing called from Start. Can we revert this?